src/coreclr/vm/threadsuspend.cpp

```diff
@@ -2309,11 +2309,7 @@ void Thread::HandleThreadAbort ()
         }
 
 #ifdef FEATURE_EH_FUNCLETS
-        if (g_isNewExceptionHandlingEnabled)
-        {
             DispatchManagedException(exceptObj);
-        }
-        else
```

Comment:
This one is tricky too. First it did one or the other, now it does both.

Reply:
DispatchManagedException does not return.

Reply:
makes sense. Fixed.